verification/cbmc_stubs/string.c

```
@@ -558,7 +558,7 @@ __CPROVER_HIDE:;
 
 #undef strlen
 
-size_t strlen(const char *s)
+size_t avocado_strlen(const char *s)
```

Cppcheck (2.19.0)

[style] The function 'avocado_strlen' is never used.

(unusedFunction)